rotkehlchen/chain/evm/decoding/interfaces.py

```diff
@@ -53,6 +53,10 @@ def enricher_rules(self) -> list[Callable]:  # pylint: disable=no-self-use
         """
         return []
 
+    def post_decoding_rules(self) -> list[Callable]:
```

Comment:
In order to be able to specify order I think this should also return an int, which should indicate priority. So all post decoding rules that have priority 0 will run first. Then those with priority 1, those with 2 etc.

Reply:
Makes sense, will add